PWGHF/TableProducer/HFTreeCreatorLbToLcPi.cxx

```diff
@@ -0,0 +1,220 @@
+DECLARE_SOA_COLUMN(CPA, cpa, float);
+DECLARE_SOA_COLUMN(CPAXY, cpaXY, float);
+DECLARE_SOA_COLUMN(Ct, ct, float);
+DECLARE_SOA_COLUMN(MCflag, mcflag, int8_t);
+// Events
+DECLARE_SOA_COLUMN(IsEventReject, isEventReject, int);
+DECLARE_SOA_COLUMN(RunNumber, runNumber, int);
+} // namespace full
+
+// put the arguments into the table
+DECLARE_SOA_TABLE(HfCandLbFull, "AOD", "HFCANDLbFull",
+                  full::RSecondaryVertex,
+                  full::DecayLength,
+                  full::DecayLengthXY,
+                  full::DecayLengthNormalised,
+                  full::DecayLengthXYNormalised,
+                  hf_cand::Chi2PCA,
+                  full::ImpactParameterNormalised0,
+                  full::PtProng0,
+                  full::PProng0,
+                  full::ImpactParameterNormalised1,
+                  full::PtProng1,
+                  full::PProng1,
+                  hf_cand::PxProng0,
+                  hf_cand::PyProng0,
+                  hf_cand::PzProng0,
+                  hf_cand::PxProng1,
+                  hf_cand::PyProng1,
+                  hf_cand::PzProng1,
+                  hf_cand::ImpactParameter0,
+                  hf_cand::ImpactParameter1,
+                  hf_cand::ErrorImpactParameter0,
+                  hf_cand::ErrorImpactParameter1,
+                  full::CandidateSelFlag,
+                  full::M,
+                  full::Pt,
+                  full::P,
+                  full::CPA,
+                  full::CPAXY,
+                  full::Ct,
+                  full::Eta,
+                  full::Phi,
+                  full::Y,
+                  full::MCflag);
+
+DECLARE_SOA_TABLE(HfCandLbFullEvents, "AOD", "HFCANDLbFullE",
+                  collision::BCId,
+                  collision::NumContrib,
+                  collision::PosX,
+                  collision::PosY,
+                  collision::PosZ,
+                  full::IsEventReject,
+                  full::RunNumber);
+
+DECLARE_SOA_TABLE(HfCandLbFullParticles, "AOD", "HFCANDLbFullP",
+                  collision::BCId,
+                  full::Pt,
+                  full::Eta,
+                  full::Phi,
+                  full::Y,
+                  full::MCflag);
+
+} // namespace o2::aod
+
+/// Writes the full information in an output TTree
+struct HfTreeCreatorLbToLcPi {
+  Produces<o2::aod::HfCandLbFull> rowCandidateFull;
+  Produces<o2::aod::HfCandLbFullEvents> rowCandidateFullEvents;
+  Produces<o2::aod::HfCandLbFullParticles> rowCandidateFullParticles;
+
+  void init(InitContext const&)
+  {
+  }
+
+  void process(aod::Collisions const& collisions,
+               aod::McCollisions const& mccollisions,
+               soa::Join<aod::HfCandLb, aod::HfCandLbMCRec, aod::HFSelLbToLcPiCandidate> const& candidates,
+               soa::Join<aod::McParticles, aod::HfCandLbMCGen> const& particles,
+               aod::BigTracksPID const& tracks)
```

Comment:
Why do you need `BigTracksPID`?

Reply:
It is currently not used. However, in anticipation for further developments (new PR soon), for when PID is implemented.